packages/aws/data_stream/cloudfront_logs/_dev/test/system/test-default-config.yml

```
@@ -9,4 +9,4 @@ data_stream:
     queue_url: "{{TF_OUTPUT_queue_url}}"
     preserve_original_event: true
 assert:
-  hit_count: 9
+  hit_count: 11
```

```suggestion
  hit_count: 12
```

I confirmed there are 12 logs in test-cloudfront.log (not counting header).